pom.xml

```diff
@@ -1987,6 +1988,8 @@
               <arg>-unchecked</arg>
               <arg>-deprecation</arg>
               <arg>-feature</arg>
+              <arg>-explaintypes</arg>
```

Comment:
explaintypes gives some better error messages in 2.11, so went ahead and turned it on